manifests/dropin_file.pp

```diff
@@ -36,7 +36,7 @@
   Optional[String]                  $source   = undef,
   Optional[Stdlib::Absolutepath]    $target   = undef,
 ) {
-  include systemd
+  include ::systemd
```

Comment:
I think that is legacy syntax at should be avoided?

Reply:
We spent a huge amount of time on Vox Pupuli (and Puppet Inc modules) to get rid of the trailing colons

Reply:
Hehe ok… I guess it's safe to deactivate that puppet-lint module then. @mcanevet your opinion?

Reply:
I'm OK with that

Reply:
I have noticed that my puppet-lint in vim (called through syntastic) often does pick up the puppet-lint module even though that particular puppet module doesn't include it. Probably because it uses the system wide puppet-lint rather than through bundler. That means I get false positives. It's not possible to ignore a check for a module that isn't installed.

```
$ grep absolute Gemfile
  #gem 'puppet-lint-absolute_classname-check',              :require => false
$ grep no-relative .puppet-lint.rc 
--no-relative_classname_inclusion-check
$ bundle exec rake lint
invalid option: --no-relative_classname_inclusion-check
```

That makes me wonder what the cleanest path forward is. Include the lint module just to disable it is also very ugly.

Reply:
My workaround was to uninstall the system-wide puppet-lint. But that is ugly as well

Reply:
Then it's a good thing it doesn't parse project settings :) It just calls puppet-lint if the filetype is puppet. Because puppet-lint finds the gem installed, it uses it. Perhaps it should first try bundle exec puppet-lint. Note that puppet-lint exists globally because bundler or rubygems creates binstubs in ~/bin.

Reply:
so in my opinion everything is fine when we remove the gem from the Gemfile? This is really legacy and should not be installed in the global ruby env, nor in any project.

Reply:
@ekohl I've often used `bundle exec vim` to use the `puppet-lint` from the bundle.

Reply:
@raphink that's a good suggestion